### DISCUSSION TITLE
Przeglądanie schema.management.azure.com

### DISCUSSION THREAD

**Discussion by kubasiatkowski** [AzureCommunityPL/group#51] [Pomoc] [answered]
Czy jest jakaś opcja na przeglądanie schema.management.azure.com bez bezpośrednich linków z dokumentacji.

Szukam schematu dla PolicySet i albo go nie ma albo moje skille w Binga są zbyt słabe :(

**Comment by kamil-mrzyglod** [comment, marked as answer]
Chodzi Ci o to? https://docs.microsoft.com/en-us/rest/api/resources/policysetdefinitions

**Reply by kubasiatkowski** [reply]
Nie,
potrzebuję coś takiego
https://schema.management.azure.com/schemas/2019-09-01/policyDefinition.json
tylko dla policySetDefinition żebym mógł w pipeline sprawdzać czy JSON w repo zgadza się ze schematem.

**Reply by kamil-mrzyglod** [reply]
Nie wiem czy jest gdzieś dostępna przegladarka online, generalnie te schemy są hostowane w tym repo - https://github.com/Azure/azure-resource-manager-schemas

**Reply by kubasiatkowski** [reply]
Dzięki, tego mi było trzeba 👍
Niestety nie ma tam tego pliku, którego potrzebuję.

**Reply by kamil-mrzyglod** [reply]
Ale tutaj chyba szukasz nie tam gdzie trzeba :) Policy Set to obiekt wewnątrz Microsoft.Authorization i inaczej nazywany jako _initiative_. Możesz go znaleźć wewnątrz tej definicji - https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2019-01-01/Microsoft.Authorization.json.

Może ewentualnie coś więcej tutaj znajdziesz - https://docs.microsoft.com/en-us/azure/governance/policy/concepts/initiative-definition-structure

**Reply by kubasiatkowski** [reply]
Dzięki, to https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2019-01-01/Microsoft.Authorization.json pomogło ale okazało się zupełnie bezużyteczne bo mogę w PS zrobić test-json z tym schematem i niezależnie od tego jakie mam śmieci w swoim pliku to dopóki parsują się jako JSON to test-json zwraca true :-/

**Comment by piotrgwiazda** [comment]
A to?

https://www.azadvertizer.net/azpolicyinitiativesadvertizer_history.html

pt., 4 wrz 2020, 13:43 użytkownik Kamil Mrzygłód <notifications@github.com>
napisał:

**Reply by kubasiatkowski** [reply]
Tylko o schemat a nie zawartość mi chodzi
